rules/windows/registry/registry_event/registry_event_defender_threat_action_modified.yml

```
@@ -0,0 +1,45 @@
+title: Windows Defender Default Threat Action Modified
+id: 5a9e1b2c-8f7d-4a1e-9b3c-0f6d7e5a4b1f
+status: experimental
+description: |
+    Detects modifications or creations of Windows Defender's default threat action settings based on severity to 'allow' or take 'no action'.
+    This is a highly suspicious configuration change that effectively disables Defender's ability to automatically mitigate threats of a certain severity level,
+    allowing malicious software to run unimpeded. An attacker might use this technique to bypass defenses before executing payloads.
+references:
+    - https://learn.microsoft.com/en-us/powershell/module/defender/set-mppreference
+    - https://learn.microsoft.com/en-us/windows-hardware/customize/desktop/unattend/security-malware-windows-defender-threatseveritydefaultaction
+    - https://research.splunk.com/endpoint/7215831c-8252-4ae3-8d43-db588e82f952
+    - https://gist.github.com/Dump-GUY/8daef859f382b895ac6fd0cf094555d2
+    - https://thedfirreport.com/2021/10/18/icedid-to-xinglocker-ransomware-in-24-hours/
+author: 'Matt Anderson (Huntress)'
+date: 2025-07-11
+tags:
+    - attack.defense-evasion
+    - attack.t1562.001
+logsource:
+    category: registry_event
+    product: windows
+detection:
+    selection_path_local:
+        TargetObject|contains: '\SOFTWARE\Microsoft\Windows Defender\Threats\ThreatSeverityDefaultAction\'
+    selection_path_gpo:
+        TargetObject|contains: '\SOFTWARE\Policies\Microsoft\Windows Defender\Threats\ThreatSeverityDefaultAction\'
+    selection_severity:
+        TargetObject|endswith:
+            - '\1' # Low severity
+            - '\2' # Moderate severity
+            - '\4' # High severity
+            - '\5' # Severe severity
+    selection_actions:
+        Details|contains:
+            - '0x00000006' # Allow
+            - '0x6'
+            - '6'
+            - '0x00000009' # NoAction
+            - '0x9'
+            - '9'
+    condition: (1 of selection_path_*) and selection_severity and selection_actions
```

```suggestion
detection:
    selection:
        TargetObject|contains: '\Microsoft\Windows Defender\Threats\ThreatSeverityDefaultAction\'
        TargetObject|endswith:
            - '\1' # Low severity
            - '\2' # Moderate severity
            - '\4' # High severity
            - '\5' # Severe severity
        Details|contains:
            - '(0x00000006)' # Allow
            - '(0x00000009)' # NoAction
```

Aren’t the details always going to be in this format (0x00000009), regardless of the how are they set, or can they differ?